src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -70,6 +70,6 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         // Keywords match amount, date and address, but does not match name
```

Comment:
```suggestion
        // Keywords match amount and date, but does not match name
```
Not sure if this testcase will still be used in the future, but we can change the description first.